src/librustc_typeck/check/writeback.rs

```diff
@@ -19,6 +19,15 @@ use syntax_pos::Span;
 
 ///////////////////////////////////////////////////////////////////////////
 // Entry point
+/// During type inference, partially inferred types are
```

Comment:
Nit: use `//`, not `///`, as this is not a doc comment

Reply:
Also, i'd prefer  a blank line

```
// Entry point
//
// During type inference, ...
```

Or perhaps move this to a module doc comment (i.e, using `//!`)?